server/tests/util/mock-automation-scheduler-server.js

```diff
@@ -6,7 +6,7 @@ const {
 const { COLLECTION_JOB_STATUS } = require('../../util/enums');
 const { default: axios } = require('axios');
 const { gql } = require('apollo-server-core');
-const { query } = require('../util/graphql-test-utilities');
+const apolloServer = require('../../graphql-server');
```

Comment:
I'm unsure I follow why make this change. Could you explain?

Reply:
This code runs during local development as part of the dev server (you can see it in server/app.js) so using a method intended exclusively for testing could potentially cause issues. I don't think there are any issues right now but it's more about preventing possible issues.

Reply:
Gotcha!